src/remote_execution.py

```diff
@@ -6,11 +6,18 @@ def get_client(ip):
     client.connect(ip, username="quilt", password="")
     return client
 
-def ssh_exec(ssh_client, cmd):
-    _, _, stderr = ssh_client.exec_command(cmd)
+def ssh_exec(ssh_client, cmd, contains_container_id=False, return_error=False):
```

Comment:
What is the point of contains_container_id?

Reply:
If I don't make the distinction between regular commands (such as package installation) and docker commands that contains container_id, then I would have to raise a SystemError for every single ssh_exec error that occur to signify that the command failed to execute. However, the current behavior is that regular ssh_exec errors don't raise SystemErrors. For my implementation, I only want to catch container_id errors only, so I added this flag to distinguish the commands.

Reply:
Why can't a regular ssh_exec error raise  System Error?

Reply:
It's not that it can't, but it was the original behavior of throttlebot to not raise such errors. There is also a case where the curl command fails to run in run_throttlebot.py, yet throttlebot continues to run with no problems.

Reply:
ok -- can you change the argument to be a bit more intuitive, like modifies_container instead of contains_container_id? I think the distinction you are making is between changing system properties on the host, vs changing it inside the container itself.

Reply:
Ok, I can implementation this change